CHANGELOG_PENDING.md

```diff
@@ -5,25 +5,26 @@ Special thanks to external contributors with PRs included in this release:
 BREAKING CHANGES:
 
 * CLI/RPC/Config
+  * [rpc] [\#2391](https://github.com/tendermint/tendermint/issues/2391) /status `result.node_info.other` became a map
 
 * Apps
-  [rpc] /status `result.node_info.other` became a map #[2391](https://github.com/tendermint/tendermint/issues/2391)
+  * [mempool] \#2310 Mempool tracks the `ResponseCheckTx.GasWanted` and enforces `ConsensusParams.BlockSize.MaxGas` on proposals.
 
 * Go API
-  * \#2310 Mempool.ReapMaxBytes -> Mempool.ReapMaxBytesMaxGas
```

Comment:
Not sure why this was deleted? This is a breaking API change, as its changed the mempool interface. The new line in apps does not note this API change.

Reply:
Fortunately for us maintainers, the `mempool` Go API is not covered by our SemVer, so we can break it as much as we want without worrying about the changelog. See https://github.com/tendermint/tendermint/#semver : 

```
To provide some stability to Tendermint users in these 0.X.X days, the MINOR version is used to signal breaking changes across a subset of the total public API. This subset includes all interfaces exposed to other processes (cli, rpc, p2p, etc.), but does not include the in-process Go APIs.
```